Bottom: 563d3a14f5f5450f582cc4e221a86ea1f4145b9b
Top:    6420506b2fb07d30117f41b412667470585f361e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-06 21:33:00 +0100

Refresh of give-the-hids-control-over-att

---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index 9be7297..87a391b 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -311,7 +311,6 @@ ghid_port_key_release_cb (GtkWidget * drawing_area, GdkEventKey * kev,
   AdjustAttachedObjects ();
   notify_crosshair_change (true);
   ghid_invalidate_all ();
-  ghid_screen_update ();
   g_idle_add (ghid_idle_cb, NULL);
   return FALSE;
 }
@@ -482,7 +481,6 @@ ghid_port_button_release_cb (GtkWidget * drawing_area,
   AdjustAttachedObjects ();
   ghid_invalidate_all ();
   notify_crosshair_change (true);
-  ghid_screen_update ();
 
   ghid_window_set_name_label (PCB->Name);
   ghid_set_status_line_label ();
